src/jquery.multiselect.js

```diff
@@ -43,60 +47,66 @@
+      htmlButtonText: false,              // (true | false) If true, then the text used for the button's label is treated as html rather than plain text.
+      htmlOptionText: false,              // (true | false) If true, then the text for option label is treated as html rather than plain text.
+      addInputNames: true,                // (true | false) If true, names are created for each option input in the multi-select.
+      disableInputsOnToggle: true,        // (true | false)
+      groupColumns: false                 // (true | false)
     },
 
-    // This method determines which element to append the menu to
-    // Uses the element provided in the options first, then looks for ui-front / dialog
-    // Otherwise appends to the body
+    /* This method determines which DOM element to append the menu to.   Determination process:
+     * 1. Look up the jQuery object, DOM element, or string selector provided in the options.
+     * 2. If nothing provided in options or lookup in #1 failed, then look for .ui-front or dialog.  (dialog case)
+     * 3. If still do not have a valid DOM element to append to, then append to the document body.
+     *
+     * NOTE:  this.element and this.document are jQuery objects per the jQuery UI widget API.
+     */
     _getAppendEl: function() {
-      var elem = this.options.appendTo;                    // jQuery object, DOM element, OR selector str.
-      if(elem) {
-        elem = elem.jquery || elem.nodeType ? $(elem) : this.document.find(elem).eq(0);  // Note that the find handles the selector case.
+      var elem = this.options.appendTo;                     // jQuery object or selector, DOM element or null.
+
+      if (elem) {                                           // NOTE: The find below handles the jQuery selector case
+        elem = !!elem.jquery ? elem : ( !!elem.nodeType ? $(elem) : this.document.find(elem).eq(0) );
       }
       if(!elem || !elem[0]) {
-        elem = this.element.closest(".ui-front, dialog");  // element is a jQuery object per http://api.jqueryui.com/jQuery.widget/
+        elem = this.element.closest(".ui-front, dialog");
       }
       if(!elem.length) {
-        elem = this.document[0].body;                      // Position at end of body.
+        elem = this.document[0].body;                      // Position at end of body.  Note that this returns a DOM element.
       }
       return elem;
     },
 
-    // Performs the initial creation of the widget
-    _create: function() {
-      var $element = this.element.hide();  // element property is a jQuery object
-      var elSelect = $element.get(0);      // This would be expected to be the underlying native select element.
+   // Performs the initial creation of the widget
+   _create: function() {
+      var $element = this.element.hide();                              // element property is a jQuery object per http://api.jqueryui.com/jQuery.widget/
```

Comment:
I don't really see a need for comments describing what a jQuery object is within a jQuery widget. I think you deleted them before too.